miden-crypto/benches/common/macros.rs

```diff
@@ -954,3 +954,81 @@ macro_rules! benchmark_word_conversions {
         }
     };
 }
+
+/// Generates comprehensive AEAD benchmarks for bytes operations.
+///
+/// This macro creates benchmarks for encryption and decryption operations
+/// using the new standardized approach with consistent data generation,
+/// throughput measurement, and reduced boilerplate.
+///
+/// # Arguments
+/// * `$aead_module` - The AEAD module name (e.g., aead_rpo)
+/// * `$group_prefix` - Human-readable prefix for benchmark group names
+///
+/// # Generated benchmarks
+/// - `benchmark_<module>_bytes` - Byte array encryption/decryption
+#[macro_export]
+macro_rules! benchmark_aead {
+    ($aead_module:ident, $group_prefix:expr) => {
+        paste::paste! {
+            /// Benchmark AEAD operations on byte arrays
+            fn [<benchmark_ $aead_module _bytes>](c: &mut Criterion) {
+                use miden_crypto::encryption::$aead_module::{Nonce, SecretKey};
+                use rand_chacha::ChaCha20Rng;
+                use rand_core::SeedableRng;
```

Comment:
Nit: traditional macro style typically uses qualified names instead of local imports, but YMMV.